### PR TITLE
Add docs for Preview Web UI

### PR DESCRIPTION
**Pull request by mosabua**
## Description

Provide minimal guidance about the new user interface to enable users to test and provide feedback and potentially help out. I am not sure about the timing for when to merge this. At a minimum we have to wait for the linked PR to be merged. Then I think we can merge this even though there will be nearly nothing to see.

## Additional context and related issues

#22869

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

File: docs/src/main/sphinx/admin/preview-web-interface.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Preview Web UI
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/preview-web-interface.md:1]: Is this the name we want to use for now .. I think yes given that the path is also `preview`.

Comment (wendigo) [reply docs/src/main/sphinx/admin/preview-web-interface.md:1]: yes, Preview Web UI sounds good

File: docs/src/main/sphinx/admin/preview-web-interface.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+In addition to the [](/admin/web-interface), Trino includes a preview version of
+a new web interface. It changes look and feel, available features, and many
+other aspects. In the future this new user interface will replace the existing
+user interface.
+
+:::{warning}
+The Preview Web UI is not suitable for production usage, and only available for
+testing and evaluation purposes. Feedback and assistance with development is
+encouraged.
```
Comment (wendigo) [inline docs/src/main/sphinx/admin/preview-web-interface.md:11]: <3

Comment (wendigo) [reply docs/src/main/sphinx/admin/preview-web-interface.md:11]: I'd add a link to the slack or whatnot to encourage colaboration

File: docs/src/main/sphinx/admin/preview-web-interface.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+:::
+
+## Activation
+
+The Preview Web UI is not available by default, and must be activated in
```
Comment (wendigo) [inline docs/src/main/sphinx/admin/preview-web-interface.md:16]: s/activated/enabled

The Preview Web UI is not exposed?

File: docs/src/main/sphinx/admin/preview-web-interface.md (hunk 2 above)
Comment (mosabua) [reply docs/src/main/sphinx/admin/preview-web-interface.md:11]: We have never done that in the docs .. but I dont see why not.

File: docs/src/main/sphinx/admin/preview-web-interface.md (hunk 3 above)
Comment (mosabua) [reply docs/src/main/sphinx/admin/preview-web-interface.md:19]: exposed sounds weird .. also style says not to use "disabled" ... 

but I did switch to "enabled"

**Pull request merged**